### ISSUE TITLE
Project Type description is entered in markdown but is not rendered in prject selection dialog

### ISSUE THREAD

**Issue by Steve-Mcl** [flowforge/flowforge#1171]
### Current Behavior

Markdown entered...
![image](https://user-images.githubusercontent.com/44235289/198996828-6a19516e-4b72-46e6-b5ea-c22c12c680b9.png)

Result...
![image](https://user-images.githubusercontent.com/44235289/198996684-f0a9f965-01e6-4c75-a6ba-f036a86a8f55.png)

### Expected Behavior

All views of project types to be rendered the same

### Steps To Reproduce

select a project type for a project without a project type

### Environment

- FlowForge version: 1.0
- Platform/OS: Windows
- Browser: Chrome

**Comment by Steve-Mcl** [comment]
@joepavitt is this be design?

I just noticed it is partially rendered (`###` is displayed in H3 but bullets are not rendered)

**Comment by joepavitt** [comment]
Not by design at all, try `-` instead of `*` for the bullet?

**Comment by knolleary** [comment]
The 'create project' page renders the list properly:

<img width="302" alt="image" src="https://user-images.githubusercontent.com/51083/199625540-378d5c2b-a7f4-4ae6-a2af-d2bd1f0fdb44.png">

So I think the only place where it appears incorrectly (at least from all the screenshots you've shared) is the 'Change Project Type' dialog you have developed in a local branch - not something we have access to.

**Comment by Steve-Mcl** [comment]
> the only place where it appears incorrectly (at least from all the screenshots you've shared) is the 'Change Project Type' dialog you have developed in a local branch - not something we have access to

No, it is an existing dialog found here: `flowforge/frontend/src/pages/project/Settings/dialogs/ChangeTypeDialog.vue`

When I was working on #595 I simply reutilised this dialog as it was already developed. 

In current V1.0, as I understand it (looking at the code in `flowforge/frontend/src/pages/project/Settings/Danger.vue`), if `!project.projectType` then this dialog will be accessible via a button labelled "Set Project Type".

**Comment by knolleary** [comment]
Ah sorry - thought this was new-new code for the Change project type item. Forgot we already had this dialog for this specific scenario.

So yes - need to figure out why it renders differently in that dialog compared to the others. I suspect it's a missing call somewhere to render the markdown.

**Comment by joepavitt** [comment]
I'll take this

**Comment by joepavitt** [comment]
Just noticed in your screenshots @Steve-Mcl - that isn't using the new `selection-tile` which is why it isn't rendering the markdown properly. I'll update it

**Comment by joepavitt** [comment]
I am unable to reproduce this though, as all new projects _require_ a project type, which then doesn't use the dialog shown.

**Comment by Steve-Mcl** [comment]
> I am unable to reproduce this though, as all new projects require a project type, which then doesn't use the dialog shown.

That is correct and expected Joe. 

When https://github.com/flowforge/flowforge/issues/595 is implemented, this dialog will be re-used and shown to users to select a different project type.

There is a _chance_ (i think) that users of old projects (before project type was introduced) that it might get shown (though I will admit I have not drilled right down into this - it may be covered by DB migrations etc already)

Either way, thanks for picking it up - will let you know if it is resolved when I get back to https://github.com/flowforge/flowforge/issues/595

**Comment by joepavitt** [comment]
In investigating, I found a race condition for the tile-selection `selected` state which I have just pushed a fix for in https://github.com/flowforge/forge-ui-components/pull/92.

I temporarily inverted the `if` condition for the showing of this option in the Project > Settings, so I can put a full fix in for this dialog - which I will do soon.
